src/workers/work.py

```diff
@@ -106,8 +108,17 @@ def skim_work(json: dict):
                     }
 
                 if return_pmids:
-                    abc_result['ab_pmid_intersection'] = str(ab['pmid_intersection'])
-                    abc_result['bc_pmid_intersection'] = str(bc['pmid_intersection'])
+                    ct_ab = dict()
+                    for pmid in sorted(ab['pmid_intersection'], key=compare, reverse=True)[:5]:
```

Comment:
would be nice to provide a top N citation count as an API post parameter instead of hardcoding 5